src/App.tsx

```diff
@@ -1,24 +1,255 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+//* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import {
+  addTodo,
+  deleteTodo,
+  getTodos,
+  patchTodo,
+} from './api/todos';
+import { Header } from './components/Header';
+import { SortType } from './types/SortType';
+import { Footer } from './components/Footer';
+import { TodoList } from './components/TodoList';
+import { Notification } from './components/Notification';
+import { ErrorType } from './types/ErrorType';
 
-const USER_ID = 0;
+const sortMethods = ['All', 'Active', 'Completed'];
+
+function creatTempTodo(userId: number, title: string): Todo {
+  return {
+    id: 0,
+    userId,
+    title,
+    completed: false,
+  };
+}
+
+function sortTodo(todos: Todo[], sortType: SortType) {
+  let newTodos = [...todos];
+
+  if (sortType === SortType.ALL) {
+    return newTodos;
+  }
+
+  newTodos = newTodos.filter(todo => {
+    switch (sortType) {
+      case SortType.ACTIVE:
+        return !todo.completed;
+
+      case SortType.COMPLETED:
+        return todo.completed;
+
+      default: return true;
+    }
+  });
+
+  return newTodos;
+}
+
+const USER_ID = 10860;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [sortType, setSortType] = useState<SortType>(SortType.ALL);
+  const [todoTitle, setTodoTitle] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [errorType, setErrorType] = useState<ErrorType>(ErrorType.NONE);
+  const [procesingTodos, SetProcesingTodos] = useState<number[]>([]);
+
+  const noneError = (): void => {
+    setTimeout(() => {
+      setErrorType(ErrorType.NONE);
+    }, 3000);
+  };
+
+  const newError = (error: ErrorType): void => {
+    setErrorType(error);
+
+    noneError();
+  };
+
+  useEffect(() => {
+    const loadTodos = async () => {
+      try {
+        const loadedTodos = await getTodos(USER_ID);
+
+        setTodos(loadedTodos);
+      } catch {
+        setErrorType(ErrorType.LOAD);
+
+        noneError();
+      }
+    };
+
+    loadTodos();
+  }, []);
+
+  const handleFilterChange = (
+    methodSort: string,
```

Comment:
My assumption use SortType enym instead of string
```suggestion
    methodSort: SortType,
```